text/0000-lts.md

```diff
@@ -0,0 +1,164 @@
+- Feature Name: lts
+- Start Date: 2018-06-21
+- RFC PR: (leave this empty)
+- Rust Issue: (leave this empty)
+
+# Summary
+[summary]: #summary
+
+Add an additional release channel for long term releases.
+
+# Motivation
+[motivation]: #motivation
+
+Rust's current release model has, overall, worked very well for us. We maintain
+a strong degree of backwards compatibility despite producing a new stable
+release very frequently. Relative to many other languages, it is very easy to
+upgrade between Rust stable releases.
+
+However, for various institutional reasons, many users do not find it
+convenient and desirable to upgrade with the frequency that Rust releases. So
+far, they have been left on their own to figure out when to actually upgrade,
+creating a situation of uncertainty and disparity. Moreover, because there are
+no norms around when an older stable stops being "supported," libraries authors
+experience a similar level of uncertainty around which versions of the compiler
+their library should be compatible with.
+
+We solve this problem by adding long-term support ("LTS") release channels,
+providing explicit support for releases beyond the latest stable.
+
+# Guide-level explanation
+[guide-level-explanation]: #guide-level-explanation
+
+## Two new channels: lts and lts-prior
+
+In addition to `stable`, `beta`, and `nightly`, we will add two additional
+release tracs:
+
+* `lts`: The most recent LTS release.
+* `lts-prior`: The second most recent LTS release.
+
+Every fourth stable/beta release, we will perform a release on both the `lts`
+and `lts-prior` channels. The **previous** stable release will be the basis for
+the new `lts` release, and the previous `lts` release will move to the
+`lts-prior` release. This is analogous to the way that the previous beta
+becomes the new stable.
+
+To keep it as easy as possible to remember which releases are lts release, we
+will begin the lts schedule so that the point version of the LTS release is
+always divisible by four. Depending on implementation schedule, that means that
+the first LTS release will probably be 1.28 or 1.32, release concurrently with
+either 1.29 or 1.33.
+
+## Level of support from the Rust project
+
+All three of the stable release channels - stable, lts, and lts-prior - will
+receive the same level of support. We will host documentation for all three
+channels and provide equivalent levels of patch release support - that is, if a
+bug justifying a patch exists in any of these channels, we will release a patch
+for that channel, just as we do today for stable. However, lts releases are
```

Comment:
I think this is a good formulation of our current support, but I might like to see it laid out a bit better for LTSes, that is, can we refine this policy too? It's basically 100% subjective at the moment.